src/components/custom/other-platforms.js

```diff
@@ -379,21 +377,22 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     onClick={onClick}
                     to="/dbot/"
                 />
-                {!is_eu_country && (
-                    <NavCard
-                        aria_label="Binary Bot"
-                        icon={() => <img src={BinaryBot} alt="" width="32" height="32" />}
-                        content={
-                            <Localize translate_text="Our classic &ldquo;drag-and-drop&rdquo; tool for creating trading bots, featuring pop-up trading charts, for advanced users." />
-                        }
-                        title={<Localize translate_text="Binary Bot" />}
-                        to="https://bot.deriv.com/"
-                        external="true"
-                        target="_blank"
-                        onClick={onClick}
-                        otherLinkProps={{ rel: 'noopener noreferrer' }}
-                    />
-                )}
+            </Flex>
+            <Flex direction="column" wrap="wrap" jc="flex-start">
+                <EmptySpace />
+                <NavCard
+                    aria_label="Binary Bot"
+                    icon={() => <img src={BinaryBot} alt="" width="32" height="32" />}
+                    content={
+                        <Localize translate_text="Our classic &ldquo;drag-and-drop&rdquo; tool for creating trading bots, featuring pop-up trading charts, for advanced users." />
+                    }
+                    title={<Localize translate_text="Binary Bot" />}
+                    to="https://bot.deriv.com/"
+                    external="true"
```

Comment:
Why we are passing `true` as a string?

Reply:
This prop is used in multiple components as string.